### PR TITLE
Add missing stream key prefixes

### PR DESCRIPTION
**Pull request by jessarcher**
#1051 fixed an issue where Predis was not applying key prefixes to stream commands, however, it missed a couple of commands that are otherwise supported by Predis.

This PR adds key prefix support for `XREVRANGE` and `XTRIM`. There are additional Redis stream commands such as `XREAD` that are missing from this list, but they don't have a corresponding class in `src/Command/Redis/` so I assume they are unsupported in Predis at this time.

### CODE REVIEW

File: src/Command/Processor/KeyPrefixProcessor.php [hunk 1]
```diff
@@ -181,9 +181,11 @@ public function __construct($prefix)
             /* ---------------- Redis 5.0 ---------------- */
             'XADD' => $prefixFirst,
             'XRANGE' => $prefixFirst,
+            'XREVRANGE' => $prefixFirst,
             'XDEL' => $prefixFirst,
             'XLEN' => $prefixFirst,
             'XACK' => $prefixFirst,
+            'XTRIM' => $prefixFirst,
```
Comment (tillkruss) [inline src/Command/Processor/KeyPrefixProcessor.php:188]: Would you mind adding tests for these? `KeyPrefixProcessorTest`

**Review by tillkruss — CHANGES_REQUESTED** [review]

File: src/Command/Processor/KeyPrefixProcessor.php (hunk 1 above)
Comment (jessarcher) [reply src/Command/Processor/KeyPrefixProcessor.php:188]: Sure, done :slightly_smiling_face:

**Review by tillkruss — APPROVED** [review]

**Comment by jessarcher** [comment]
@tillkruss Not urgently, but it would be appreciated when you have the time.

**Comment by tillkruss** [comment]
@vladvildanov Can you add these commands to `ClusterStrategy` as well, like you did in #1235?

**Comment by vladvildanov** [comment]
@tillkruss Did it within my PR 👌

**Comment by tillkruss** [comment]
Thanks @vladvildanov ❤️‍🔥

**Pull request merged**